### PR TITLE
User organization details's History modal tests

### PR DESCRIPTION
**Pull request by Nightrider0098**
## Describe your changes
Add Tests cases for History modal in Organization Details  

## Issue ticket number and link
https://github.com/stakwork/sphinx-tribes-frontend/issues/193
https://community.sphinx.chat/bounty/1441

## Type of change
Test Case

### CODE REVIEW

**Comment by Nightrider0098** [comment]
image

**Comment by ecurrencyhodler** [comment]
Paid 300k sats! https://community.sphinx.chat/bounty/1441

File: src/people/widgetViews/OrganizationDetails.tsx [hunk 1]
```diff
@@ -136,7 +136,7 @@ const OrganizationDetails = (props: {
     async (user: any) => {
       const pubkey = user.owner_pubkey;
       if (uuid && pubkey) {
-        const userRoles = await await main.getUserRoles(uuid, pubkey);
+        const userRoles = await main.getUserRoles(uuid, pubkey);
```
Comment (kevkevinpal) [inline src/people/widgetViews/OrganizationDetails.tsx:139]: my only thought is that doing this broke staging? @elraphty what do you think

Comment (ecurrencyhodler) [reply src/people/widgetViews/OrganizationDetails.tsx:139]: Were you able to find out what was?

**Pull request merged**